### PR TITLE
Added travis ci support for node 7

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
+  - 7
```
Comment (andywer) [inline .travis.yml]: Since even node versions are the LTS releases: Replace `latest` by `7` and skip the `5` altogether?

Comment (amilajack) [inline .travis.yml]: Good idea

**Comment by amilajack** [comment]
Fixed

**Comment by andywer** [comment]
Ahh, I meant "replace '7' with 'latest'". But nevermind, explicit versions are good ;)

Thanks for contributing!

**Pull request merged**